src/components/Authorization/Authorization.jsx

```diff
@@ -32,114 +33,168 @@ export const Authorization = ({primary}) => {
+          userEmail: email,
+          userPassword: password,
+        }),
+      );
+    } catch (error) {
+      setEmailError("Неверный логин или пароль");
+      setPasswordError("Неверный логин или пароль");
+    }
+  };
+  const onEmailChange = (event) => {
+    setEmailInputText(event.target.value);
     setEmail(event.target.value);
     setEmailError("");
   };
   const onPasswordChange = (event) => {
+    setPasswordInputText(event.target.value);
     setPassword(event.target.value);
     setPasswordError("");
   };
-  const validateEmail = (email) => {
-    const emailRegex = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
-    return emailRegex.test(email);
-  };
-  const validatePassword = (password) => {
-    if (typeof password === 'string') {
-      return password.trim() !== '' && password.length >= 8;
-    } else {
-      return false;
-    }
-  };
   const togglePasswordVisibility = () => {
     setPasswordVisible(!passwordVisible);
   };
   return (
-    <div className='wrapper'>
+    <div className="wrapper">
       <Header />
-      <div className='wrapper-m'>
-        <div className='block-auth' aria-label='диалоговое окно для авторизации'>
-          <img src={imageCard} alt="картинка с сине-голубым оттенком" className='imageCard' />
-          <div className='dialog-window'>
-            <h1 className='dialog-title'>Mы рады вас видеть</h1>
+      <div className="wrapper-m">
+        <div
+          className="block-auth"
+          aria-label="диалоговое окно для авторизации"
+        >
+          <img
+            src={imageCard}
+            alt="картинка с сине-голубым оттенком"
+            className="imageCard"
+          />
+          <div className="dialog-window">
+            <h1 className="dialog-title">Mы рады вас видеть</h1>
+
             <form className="dialog-content" onSubmit={handleSubmit}>
-              <p className={"error " + (emailError ? "error-text" : "")}>{emailError}</p>
-              <input className='input-field' type="email"
-                          id="email"
-                          name="email"
-                          placeholder="Логин"
-                          value={email}
-                          onChange={onEmailChange}
-                          style={{borderColor: emailError ? "red" : "##282828", borderWidth: '1px'}}/>
-              <p className={"error " + (passwordError ? "error-text" : "")}>{passwordError}</p>
-              <input className='input-field' type={passwordVisible ? 'text' : 'password'}
-                          id="pass"
-                          name="password"
-                          minLength={8}
-                          required
-                          value={password}
-                          placeholder="Пароль"
-                          onChange={onPasswordChange}
-                          style={{borderColor: passwordError ? "red" : "#282828", borderWidth: '1px'}}/>
+              <p className={"error " + (emailError ? "error-text" : "")}>
```

Comment:
Don't use ternary operators like this. We just can do the same with this code:
```
emailError && "error-text"
```
But better if we just show this HTML element with ternary:
```
            {emailError  &&  <p className={"error " + "error-text"}>}
```

Reply:
fix